semconvgen/generator.go

```diff
@@ -379,6 +388,30 @@ var capitalizations = []string{
 	"OTel",
 }
 
+func capitalizations(cfg config) ([]string, error) {
+	c := append([]string(nil), staticCapitalizations...)
+
+	if cfg.capitalizationsPath != "" {
+		file, err := os.Open(cfg.capitalizationsPath)
+		if err != nil {
+			return nil, fmt.Errorf("unable to open capitalizations file: %w", err)
+		}
+		defer file.Close()
+
+		scanner := bufio.NewScanner(file)
+		for scanner.Scan() {
+			capitalization := strings.TrimSpace(scanner.Text())
+			c = append(c, capitalization)
+		}
```

Comment:
Implemented this as plaintext, new line separated, but if folks think yaml/json (or something else) would be better I'm open to suggestions.